src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentForm.java

```diff
@@ -439,8 +478,11 @@ public boolean isDocEnroute() {
         return ObjectUtils.isNotNull(workflowDocument) && workflowDocument.isEnroute();
     }
     
-    public String getContractWarningMessage() {
-        return getConfigurationService().getPropertyValueAsString(CUPurapKeyConstants.MESSAGE_IWNT_CONFIRM_CREATE_REQ);
+    public String getContractWarningMessage(String docType) {
+        String message = getConfigurationService().getPropertyValueAsString(CUPurapKeyConstants.MESSAGE_IWNT_CONFIRM_CREATE_REQ_OR_DV);
+        String messageFormattedText = MessageFormat.format(message,
+                docType);
```

Comment:
Is there a reason why we need two local variables here? 
Meaning, is there a reason why this cannot be consolidated into :
`return MessageFormat.format(getConfigurationService().getPropertyValueAsString(CUPurapKeyConstants.MESSAGE_IWNT_CONFIRM_CREATE_REQ_OR_DV), docType);`

Reply:
updated